src-gui/src/renderer/components/pages/history/table/HistoryRowExpanded.tsx

```diff
@@ -85,7 +89,7 @@ export default function HistoryRowExpanded({
             <TableRow>
               <TableCell>Provider Address</TableCell>
               <TableCell>
-                <Box>{swap.seller.addresses.join(", ")}</Box>
+                <Box>{swap.seller.addresses.map(addr => <ClickableAddress address={addr} noIcon />)}</Box>
```

Comment:
We need to add a [key](https://legacy.reactjs.org/docs/lists-and-keys.html) to the ClickableAddress inside the map callback to make eslint happy and avoid unnecessary re-renders.